it-test/src/test/java/org/opennms/plugins/cloud/ittest/EndToEndIt.java

```diff
@@ -63,10 +69,16 @@ public class EndToEndIt {
     private KarafShell opennmsShell;
     private KarafShell sentinelShell;
 
+    private static String parentVersion;
+
     @BeforeClass
-    public static void beforeAll() {
+    public static void beforeAll() throws IOException, XmlPullParserException {
+        MavenXpp3Reader mavenXpp3Reader = new MavenXpp3Reader();
+        Model model = mavenXpp3Reader.read(new FileReader("pom.xml"));
+        parentVersion = model.getParent().getVersion();
         environment = new DockerComposeContainer<>(new File("src/test/resources/docker-compose.yaml"))
                 .withEnv("USER_HOME", System.getProperty("user.home"))
+                .withEnv("PARENT_VERSION", parentVersion)
```

Comment:
nit: naming: why parent version? I wonder if it shouldn't be "PLUGIN_VERSION"?

Reply:
Just because the method to get it is called model.getParent().getVersion()

Reply:
understand. But the content of the variable is the version of the plugin. I would name it accordingly.